notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>Please preserve output of these cells for demonstration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>